src/screens/HomeScreen.js

```diff
@@ -1,8 +1,37 @@
-import React from 'react';
-import {Text} from 'react-native';
+import React, {useEffect, useState} from 'react';
+import {StatusBar, StyleSheet, ScrollView} from 'react-native';
+
+import ImageToolbar from '../components/ImageToolbar';
 
 const HomeScreen = () => {
-  return <Text>Hi, Welcome Home</Text>;
+  const [photos, setPhotos] = useState(null);
+
+  useEffect(() => {
+    fetch(
+      'https://api.unsplash.com/photos/?client_id=Pfsu-9Fcu8eaLxSJSYQlXmxoaS3X90KHGJ8AlSJndnE',
+    )
+      .then(response => response.json())
+      .then(data => {
+        setPhotos(data);
+      });
+  }, []);
+
+  return (
+    <ScrollView style={styles.scrollView}>
```

Comment:
@geekiam23 We can make this scrollable list better by using FlatList instead of ScrollView. ScrollViews are good for a limited number of items, but our use case is calling for a huge list of items and ScrollView will have performance issues

Reply:
@geekiam23 Here is a good article from React Native explaining the performance difference https://reactnative.dev/docs/scrollview#:~:text=vs,this%20has%20a%20performance%20downside.&text=FlatList%20renders%20items%20lazily%2C%20when,save%20memory%20and%20processing%20time.